app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None | bool:
+    command = command.split(" ")[1:]
+    if command[0] != command[1]:
```

Comment:
It is better to unpuck your commend[0] and command[1] into a viriables fot better understanding . You can do it in one line